### PR TITLE
upgrade guzzle version to 6.3

### PR DESCRIPTION
**Pull request by r-miezhelis**

### CODE REVIEW

**Comment by r-miezhelis** [comment]
Hey!
I have a problem with phpunit.
I see the next exception
`1) Gengo\Tests\JobTest::testRetrievesTheCommentThreadForAJob
GuzzleHttp\Exception\ServerException: Server error: `GET https://api.sandbox.gengo.com/v2/translate/job/2894429/comments?ts=1539080401&api_key=pubkeyfortests&api_sig=43e63fc07dce328dda1155f08fdbba9b82546a99` resulted in a `500 Internal Server Error` response:
{"err":{"msg":"pq: invalid input value for enum mg.contexttype: \"flag_unsuitable\""},"opstat":"error"}`
And the same exceptions i have on original branch.
What could be the reason?

**Comment by ryanannatuvasquez** [comment]
@r-miezhelis thankyou for contributing! can you add more description why you want to upgrade current guzzle version.

**Comment by r-miezhelis** [comment]
Hello @ryanannatuvasquez! Thanks for your response.
Because this is actual version of guzzle.
Version 5 and version 6 are incompatible.
For some libraries i use i need to have guzzle ^6.3.

**Comment by ryanannatuvasquez** [comment]
@r-miezhelis I fixed the error you encountered in the test, but still you have error for linting, kindly addressed before we review the actual pull request

**Comment by r-miezhelis** [comment]
Done!

File: src/Client.php [hunk 1]
```diff
@@ -206,14 +206,10 @@ private static function _request($url, $method, array $params, array $files = []
...
-                    $options['body'] = $params;
-                    foreach ($files as $key => $file) {
-                        $options['body'][$key] = new PostFile($key, fopen($file, 'r'));
-                    }
-                    self::$response = self::$http->post($url, $options);
+                    self::$response = self::$http->post($url, self::getPostOptions($files, $params));
                     break;
                 case 'PUT':
-                    $options['body'] = $params;
+                    $options[RequestOptions::FORM_PARAMS] = $params;
```
Comment (ryanannatuvasquez) [inline src/Client.php]: `RequestOptions::FORM_PARAMS` is `form_params`, im expecting this to be the same or use `RequestOptions::BODY` which is `body` https://github.com/guzzle/guzzle/blob/003757426cdd50fbad1c34cdbcc8a55f70a4fe8b/src/RequestOptions.php#L49

**Comment by r-miezhelis** [comment]
Hey guys, any updates?

**Comment by ryanannatuvasquez** [comment]
@r-miezhelis https://packagist.org/packages/gengo/gengo#3.1.1 is now available! thankyou very much 🙇

**Comment by r-miezhelis** [comment]
Thank you!

**Pull request merged**